experiments/helper.py

```diff
@@ -84,6 +84,16 @@ def make_efficient_split(data_args, split_name, ner_tasks):
     return dataset
 
 
+def filter_by_length(row):
+    """
+    Removes examples that are too short
+    :param example:    the example to check
+    :return:
+    """
+    if 10 < len(str(row["input"])):
```

Comment:
This might lead to different results on LEXTREME if we rerun the models. In some datasets we didn't filter out these very small ones before.

Reply:
But we can still keep filtering completely empty examples...

Reply:
We do filter completely empty examples directly in the template_SLTK.py (see line 127-129). This function was not used.